tests/actions_self_hosted_runners.rs

```diff
@@ -103,8 +103,8 @@ async fn test_context(
     resp_body: Option<impl Serialize>,
 ) -> TestContext {
     let template = match resp_body {
-        Some(resp_body) => ResponseTemplate::new(response_code).set_body_json(resp_body),
-        None => ResponseTemplate::new(response_code),
+        Some(resp_body) => ResponseTemplate::new(response_code.as_u16()).set_body_json(resp_body),
+        None => ResponseTemplate::new(response_code.as_u16()),
```

Comment:
wiremock with http@1.0 is not published yet. I simply convert `StatusCode (http@1.0) -> u16 -> StatusCode (http@0.2)`